package-lock.json

```diff
@@ -4798,6 +4798,16 @@
       "integrity": "sha512-jDctJ/IVQbZoJykoeHbhXpOlNBqGNcwXJKJog42E5HDPUwQTSdjCHdihjj0DlnheQ7blbT6dHOafNAiS8ooQKA==",
       "dev": true
     },
+    "bindings": {
```

Comment:
The `package-lock.json` changes should probably not be part of this pull request since there where no changes to the `package.json`. So there is no obvious reason why this would be needed.